pack/tcu/icc_encounter.json

```diff
@@ -0,0 +1,563 @@
+		"double_sided": true,
+		"encounter_code": "music_of_the_damned",
+		"encounter_position": 3,
+		"faction_code": "mythos",
+		"illustrator": "Joshua Cairós",
+		"name": "Hangman's Hill",
+		"subname": "Where It All Ends",
+		"pack_code": "icc",
+		"position": 302,
+		"quantity": 1,
+		"shroud": 2,
+		"text": "[action]: Search the encounter deck and discard pile for a [[Witch]] enemy and draw it. Then, move all breaches from Hangman's Hill to the current act. Shuffle the encounter deck.",
+		"traits": "Arkham.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "As you approach the manor of the Lodge once more, you find it dark and empty. It seems that its members are still laying low, at least for the time being.",
+		"clues": 1,
+		"code": "05303",
+		"double_sided": true,
+		"encounter_code": "music_of_the_damned",
+		"encounter_position": 4,
+		"faction_code": "mythos",
+		"illustrator": "Jokubas Uogintas",
+		"name": "Silver Twilight Lodge",
+		"subname": "Shrouded In Mystery",
+		"pack_code": "icc",
+		"position": 303,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "[free] Take 1 horror: Move 1 breach from Silver Twilight Lodge to the current act.",
+		"traits": "Arkham.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "As you approach Hangman’s Hill once more, you find it devoid of any activity-from the living or otherwise. It seems the coven has left this place behind, at least for the time being.",
+		"clues": 1,
+		"code": "05304",
+		"double_sided": true,
+		"encounter_code": "secrets_of_the_universe",
+		"encounter_position": 3,
+		"faction_code": "mythos",
+		"illustrator": "Joshua Cairós",
+		"name": "Hangman's Hill",
+		"subname": "Shrouded In Mystery",
+		"pack_code": "icc",
+		"position": 304,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "[free] Take 1 damage: Move 1 breach from Hangman's Hill to the current act.",
+		"traits": "Arkham.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "As you approach the manor of the Lodge once more, bright light shines from behind its windows. You can hear faint chanting from within.",
+		"clues": 0,
+		"code": "05305",
+		"double_sided": true,
+		"encounter_code": "secrets_of_the_universe",
+		"encounter_position": 4,
+		"faction_code": "mythos",
+		"illustrator": "Jokubas Uogintas",
+		"name": "Silver Twilight Lodge",
+		"subname": "Where It All Ends",
+		"pack_code": "icc",
+		"position": 305,
+		"quantity": 1,
+		"shroud": 2,
+		"text": "[action]: Search the encounter deck and discard pile for a [[Silver Twilight]] enemy and draw it. Then, move all breaches from Silver Twilight Lodge to the current act. Shuffle the encounter deck.",
+		"traits": "Arkham.",
+		"type_code": "location"
+	},
+	{
+		"code": "05306",
+		"encounter_code": "in_the_clutches_of_chaos",
+		"encounter_position": 15,
+		"faction_code": "mythos",
+		"flavor": "You look upon the face of true chaos. Everything you have ever known, everything you have ever loved-it is all meaningless by comparison.",
```

Comment:
Please put spaces either side of "loved-it": "loved - it" (We opt for dash with spaces rather than em dash on the db.)